scripts/changed-files

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
This doesn't feel very Windows friendly

Reply:
It ain't. Are you developing under Windows?

Reply:
Occasionally! I try to keep things shell/os-agnostic in my own projects. I dislike the idea of having a barrier to contributing based on OS when I could structure things to be more compatible

Reply:
Yeah. I try to be as friendly as possible, but something I've learn along the years is that there is a limit on how much out of shape one should bend for the remote possibility that somebody with a specific setup might potentially want to contribute. My rule of thumb is: optimize for the comfort of who are working right now on the thing, and don't actively be a jerk for the rest.